.github/workflows/master-to-relrc.yml

```diff
@@ -0,0 +1,25 @@
+name: Create rel/rc pull request
+on:
+  push:
+    branches:
+      - master
+jobs:
+  relrc-promotion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: rel/rc
+      - name: Merge latest master to rel/rc
+        run: |
+          git fetch origin master:master
+          git reset --hard master
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: format('chore', ':', 'merge latest to rel/rc')
+          title: Merge latest to rel/rc
+          body: |
+            chore: merge latest master to rel/rc
+            BG-000000
+          branch: master-to-rel/rc
```

Comment:
Depends on preference from the team. If there are merge conflicts from master to rel/rc with master being the source of truth, we can make the merge conflicts visible and resolve them via a `master-to-rel/rc` branch since master doesn't need to be updated. Also allows us to go in and fix `rc` tags if needed since the merge doesn't know that it isn't supposed to override `rc` tags in package.json files.

Reply:
That makes sense and looks good to me!